app/views/shared/404.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.session do
+  json.success false
+  json.message 'Authentication error'
```

Comment:
Authentication error...`401 unauthorized` code better than `404 not found`. What do you think?